### ISSUE TITLE
Holding Ctrl doesn't work anymore with chrome

### ISSUE THREAD

**Issue by glyh** [hyprwm/Hyprland#6162]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit cba1ade848feac44b2eda677503900639581c3f4  (props: bump version to 0.40.0).
Date: Sat May 4 15:42:32 2024
Tag: v0.40.0, commits: 4606

flags: (if any)


System Information:
System name: Linux
Node name: mechrevo
Release: 6.9.1-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Fri, 17 May 2024 16:56:38 +0000


GPU information: 
00:02.0 VGA compatible controller [0300]: Intel Corporation Alder Lake-P GT1 [UHD Graphics] [8086:46a3] (rev 0c) (prog-if 00 [VGA controller])
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation AD107M [GeForce RTX 4060 Max-Q / Mobile] [10de:28a0] (rev a1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:


```

</details>

### Bug or Regression?

Regression

### Description

I used to be able to hold ctrl and press a bunch of hotkeys on chrome but it won't work any more. 

### How to reproduce

1. `google-chrome-stable --enable-features=UseOzonePlatform --ozone-platform=wayland --enable-wayland-ime`
2. have focus not on the address bar
3. Hold Ctrl and press L to focus address
4. Keep holding Ctrl and press V, clipboard is not passed, but rather a `v` occured.

### Crash reports, logs, images, videos

This is not a crash. Also opened on thorium

**Comment by vaxerski** [comment]
can you test on git?

**Comment by FakeMichau** [comment]
I've seen this issue on Firefox and I think some other places as well. 
It's fixed on git whereas 0.40.0 has the issue.

**Comment by glyh** [comment]
I can confirm this is fixed on git. Thanks for the information!
